tools/peptidegenomiccoordinate/.shed.yml

```diff
@@ -0,0 +1,11 @@
+categories: [Proteomics]
+description: Gets genomic coordinate of peptides based on the information in mzsqlite and genomic mapping sqlite files
+long_description: |
+    This program loads two sqlite databases (mzsqlite and genomic mapping sqlite files) and calculates the genomic coordinates of the peptides provided as input. This outputs bed file with genomic coordinate of peptides.
+    
+    Input: Peptide list file, mzsqlite sqlite DB file, and genomic mapping sqlite DB file
+    Output: Tabular BED file with all the columns
+   
+name: peptidegenomiccoordinate
```

Comment:
can we rename the folder and the tool to `peptide_genomic_coordinate`

@pravs3683 I need to more time to review the python code do you have any other reviewer?